src/Sulu/Bundle/MediaBundle/DependencyInjection/Configuration.php

```diff
@@ -224,6 +227,21 @@ private function addStorageSection(ArrayNodeDefinition $node)
                 ->end();
         }
 
+        if (class_exists(AzureBlobStorageAdapter::class)) {
+            $storages[] = self::STORAGE_AZURE_BLOB;
+
+            $storagesNode
+                ->arrayNode(self::STORAGE_AZURE_BLOB)
+                ->addDefaultsIfNotSet()
```

Comment:
Can you indent this lines to match the tree like in the other configuraitons

Reply:
Done!

Reply:
This line should also be indented. So L235-L241 should be +4 spaces. See S3 example.

Reply:
I didn't realize 😅 , I think it's already

Reply:
Thank you!